spec/registry/gcp/dataproc/facets/GcpDataprocSparkRunFacet.json

```diff
@@ -50,6 +50,10 @@
               "description": "The name of the query node in the executed Spark Plan. Often used to describe the command being executed.",
               "type": "string"
             },
+            "runType": {
```

Comment:
How can `job` be a `runType`? Perhaps I am biased by OpenLineage semantics of job and its runs. In this case `runType` equal to `job` makes little sense.

Reply:
I have renamed the property to `jobType`